gson/src/main/java/com/google/gson/GsonBuilder.java

```diff
@@ -337,7 +338,6 @@ public GsonBuilder enableComplexMapKeySerialization() {
    *
    * <p>In general using inner classes with Gson should be avoided; they should be converted to {@code static}
    * nested classes if possible.
-   *
```

Comment:
Really minor but maybe add back this empty line for consistency (I think you removed it by accident)